buildlib/azure-pipelines-pr.yml

```diff
@@ -125,7 +129,7 @@ stages:
             rhel82:
               CONTAINER: rhel82
         container: $[ variables['CONTAINER'] ]
-        timeoutInMinutes: 360
+        timeoutInMinutes: 120
```

Comment:
Cuda build can be more that 2h

Reply:
updated to 3h (though currently the build takes about 1-1.5h, better keep it on safe side)